openmdao/solvers/linear/tests/test_petsc_ksp.py

```diff
@@ -508,8 +508,8 @@ def test_feature_maxiter(self):
         of = ['obj']
 
         J = prob.compute_totals(of=of, wrt=wrt, return_format='flat_dict')
-        assert_rel_error(self, J['obj', 'z'][0][0], 9.2654054431, .00001)
-        assert_rel_error(self, J['obj', 'z'][0][1], 1.87246623559, .00001)
+        assert_rel_error(self, J['obj', 'z'][0][0], 4.93218027, .00001)
```

Comment:
Note to reviewers: Bret and I looked at this test, and changing the numbers if appropriate here.